dashboard/app/controllers/concerns/pd/workshop_filters.rb

```diff
@@ -85,6 +85,17 @@ def filter_workshops(workshops)
       workshops = workshops.where(subject: params[:subject]) if params[:subject]
 
       if current_user.permission?(UserPermission::WORKSHOP_ADMIN)
+        if params[:virtual]
```

Comment:
small efficiency improvement: if we put the virtual filter last we will have fewer workshops to filter through